### PR TITLE
Add LabeledInput component

### PR DESCRIPTION
**Pull request by lajanzen**
Create labeled input field for login & register forms

### CODE REVIEW

**Review by Maurizio-F — APPROVED** [review]
Good job! 👍

**Review by Dev-dfm — APPROVED** [review]
lgtm 👍

File: src/app/components/LabeledInput/LabeledInput.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+function LabeledInput({
+  label,
+  value,
+  type,
+  required,
+  onChange,
+}: LabeledInputProps): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <label className={styles.container__label}>{label}</label>
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.tsx]: Not a big issue, but you could connect the label with the input (if the input is a child of the label).
This makes it possible to click the label to select the input field.

**Review by lmachens — APPROVED** [review]
gj

File: src/app/components/LabeledInput/LabeledInput.tsx (hunk 1 above)
Comment (lajanzen) [reply src/app/components/LabeledInput/LabeledInput.tsx]: Good point, thanks! I tried to connect them - maybe you could recheck?

File: src/app/components/LabeledInput/LabeledInput.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+export default {
+  title: "Component/LabeledInput",
+  component: "LabeledInput",
+};
+
+export const Email = (): JSX.Element => (
+  <LabeledInput
+    label="E-Mail"
+    id="mail"
+    htmlFor="mail"
```
Comment (lmachens) [inline src/app/components/LabeledInput/LabeledInput.stories.tsx:13]: I think these are always the same? Maybe it's enough to just add `id`?

**Review by lmachens — APPROVED** [review]
This works well. An alternative is to cascade `input` in `label` (no `id` is required)

File: src/app/components/LabeledInput/LabeledInput.stories.tsx (hunk 2 above)
Comment (lajanzen) [reply src/app/components/LabeledInput/LabeledInput.stories.tsx:13]: I first cascaded it, but then it was harder to style it in CSS.
If I just add id, are they still connected?

Comment (lmachens) [reply src/app/components/LabeledInput/LabeledInput.stories.tsx:13]: You can say `htmlFor={id}` to connect it :)

**Pull request merged**